collectors/o365/lib/o365_mgmnt/o365management.js

```diff
@@ -67,25 +67,7 @@ class O365Management extends msRestAzure.AzureServiceClient {
         }
     }
 
-    // standard request handler
-    requestHandler(httpRequest){
```

Comment:
I suggest we keep existing functions as is and add a new one, like, `getNextPageUrl(nextPageUrl, options)` helper function which simply retrieves the next page. This approach gives the called full control over pagination and then we can have the actual page listing inside `o365_collector.js`. We may want to have some pagination configuration (for example, how many pages we'd like to process before sending data to ingest) which we'll be able to add via template params or function configuration.